packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -368,6 +381,17 @@ watch(
 	{ immediate: true }
 );
 
+async function saveAsNewModel(model: Model) {
+	if (model) {
+		const newModel = await createModel(model);
+		if (!newModel) {
+			return;
+		}
+		amr.value = newModel;
+		createOutput(newModel);
+	}
+}
+
```

Comment:
the `save-asset-modal` already creates a new model and adds it to your project use `save-asset.ts`, I don't think there is a need to create it here.